.github/workflows/publish-example-images.yaml

```
@@ -33,12 +32,15 @@ jobs:
           - component-name: tf-smoke
             dockerfile: examples/tensorflow/tf_sample/Dockerfile
           - component-name: pytorch-dist-sendrecv-test
+            platforms: linux/amd64,linux/arm64
             dockerfile: examples/pytorch/smoke-dist/Dockerfile
           - component-name: pytorch-elastic-example-imagenet
+            platforms: linux/amd64,linux/arm64
             dockerfile: examples/pytorch/elastic/imagenet/Dockerfile
           - component-name: pytorch-elastic-example-echo
             dockerfile: examples/pytorch/elastic/echo/Dockerfile
```

```suggestion
          - component-name: pytorch-elastic-example-echo
            dockerfile: examples/pytorch/elastic/echo/Dockerfile
            platforms: linux/amd64,linux/arm64
```

Could you also address this image?

---

Sorry yes, I overlooked this